src/observer/management/commands/update_listener.py

```diff
@@ -0,0 +1,46 @@
+import sys, json
```

Comment:
I'd keep an eye on this listener modules to make them extractable into a `bus-sdk-python` similar to https://github.com/elifesciences/bus-sdk-php as we have seen them pop up at least in `observer`, `elife-metrics`, `bot-lax-adaptor`. There is some machinery around the catching of exceptions, deletion of used messages and seamless New Relic monitoring that could be shared by projects.

Reply:
or add them to https://github.com/elifesciences/bus-sdk-python

Reply:
That's the perfect place!

Reply:
I'll open tickets to add that functionality and integration into the python projects.